src/app/workspace/resource/representation/upload/upload.component.ts

```diff
@@ -46,13 +46,13 @@ export class UploadComponent implements OnInit {
     thumbnailUrl: string | SafeUrl;
 
     allowedFileTypes: string[];
-    // todo: maybe we can use this list to display which file format is allowed to
-    supportedImageTypes = ['image/jpeg', 'image/jp2', 'image/tiff', 'image/tiff-fx', 'image/png'];
-    supportedDocumentTypes = ['application/pdf'];
-    supportedAudioTypes = ['audio/mpeg'];
-    supportedVideoTypes = ['video/mp4'];
-    supportedArchiveTypes = ['application/zip', 'application/x-tar', 'application/gzip'];
-    supportedTextTypes = ['application/csv', 'application/xml', 'text/csv', 'text/plain', 'text/xml'];
+
+    supportedAudioTypes = ['mp3', 'wav'];
+    supportedArchiveTypes = ['zip', 'tar', 'gz', 'gzip', '7z', 'tgz', 'z'];
```

Comment:
For all supported types: Form what the order of the types is deriving from? For my taste it would be nice to have them ordered alphabetically, hard coded and/or via .sort()'

Reply:
it's the order that's defined in the API so that I could make sure that I included everything easily but I can sort them alphabetically

https://github.com/dasch-swiss/dsp-api/blob/main/sipi/scripts/file_info.lua

Reply:
done in [635ae3a](https://github.com/dasch-swiss/dsp-app/pull/808/commits/635ae3a38c575c9b6ecc923fe4a096459eae349f)